outlierdetect/outlierdetect.py

```diff
@@ -238,21 +240,23 @@ def _normalize(self, value_dict):
 
 ########################################## Helper functions ########################################
 
-def _normalize_counts(counts, val=1):
+def _normalize_counts(counts, val=1, percent_format=False):
```

Comment:
I don't see any changes to the call to `_normalize_counts` in the PR. Since this is a private method (a single underscore before a method name means that it is not imported when using it as a module), what is the point of this?